src/components/playerdash/elo-graph.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Link } from 'react-router';
+import FirebaseLib from '../../utils/FirebaseLib.js';
+var Chart = require('react-google-charts').Chart;
+
+
+module.exports = React.createClass( {
+
+  getInitialState() {
+    return {
+      AnnotationChart: {
+        rows:[],
+        columns:[],
+        chartType: ""
+      }
+    }
+  },
+
+  componentWillMount() {
+    var AnnotationChart =  {
+      rows : this.props.graph,
+      columns : [
+      {
+        label : "date time",
+        type: "datetime"
+      },
+      {
+        label : "Elo Rating",
+        type: "number"
+      }
+
+      ],
+      options : {title: "Elo Rating", hAxis: {title: 'Date' }, vAxis: {title: 'Elo Rating', format:'####'}},
+      chartType : "AnnotationChart",
+      div_id: "elo_line_graph"
+    };
+
+    this.setState({
+      'AnnotationChart': AnnotationChart
+    });
+  },
+
+  render() {
+    return (
+      <Chart chartType={this.state.AnnotationChart.chartType} width={"100%"} height={"600px"} rows={this.props.graph} columns={this.state.AnnotationChart.columns} options = {this.state.AnnotationChart.options} graph_id={this.state.AnnotationChart.div_id}  />
+      );
+  }
+});
```

Comment:
I think this file can be simplified a lot. Since the only data this module is concerned with is passed as props from the parent we don't need to worry about state. State is always handled by the parent and if the parent state changes those changes will be passed to the child and re-rendered.

We also don't need to worry about componentWillMount in this instance again since there's nothing special that needs to happen then and data will already be available via props. 

So I think the best place to setup the data object for the graph is in render. I think it makes sense to keep it declared as a variable because it makes it really easy to read without the added jsx markup.

I've broken the `<Chart...` jsx into multiple lines. I have a guide setup at 80 charactors in my editor and use it as a guide for when I should wrap things. I'm not hard and fast about 80 chars but if it's more readable wrapping after that then I'll switch to a wrapped syntax.

Also just a change to the import syntax of `Chart` - I have checked multiple times if any of this was causing my problem with the chart module and had the same error with it written either way. And I've removed a couple of unused imports.

```jsx
import React from 'react';
import { Chart } from 'react-google-charts';

module.exports = React.createClass( {

  render() {
    let AnnotationChart =  {
      rows : this.props.graph,
      columns : [
        { label : "date time", type: "datetime" },
        { label : "Elo Rating", type: "number" }
      ],
      options : {
        title: "Elo Rating",
        hAxis: {title: 'Date' },
        vAxis: {title: 'Elo Rating', format:'####'}
      },
      chartType : "AnnotationChart",
      div_id: "elo_line_graph"
    };

    return (
      <Chart chartType={AnnotationChart.chartType}
        width={"100%"}
        height={"600px"}
        rows={this.props.graph}
        columns={AnnotationChart.columns}
        options = {AnnotationChart.options}
        graph_id={AnnotationChart.div_id}
      />
    );
  }
});
```